### ISSUE TITLE
Glossary terms nested inside vertices don't work

### ISSUE THREAD

**Issue by danieleades** [danieleades/sphinx-graph#42]
Glossary terms nested inside vertices don't work. It's most obvious when using the `latexpdf` builder (seems to just fail silently when building html)

```
Exception occurred:
  File "/home/daniel.eades/Code/python/sphinx-graph/.venv/lib/python3.10/site-packages/docutils/nodes.py", line 652, in __getitem__
    return self.attributes[key]
KeyError: 'refdocname'
```

**Comment by danieleades** [comment]
see https://stackoverflow.com/questions/74575915/custom-sphinx-directive-fails-when-the-content-contains-a-nested-glossary-term

**Comment by danieleades** [comment]
note that this bug also impacts the 'todo extension' sphinx tutorial on which this extension is based.
see https://github.com/sphinx-doc/sphinx/issues/10992

**Comment by danieleades** [comment]
fixed in #47
